package/helm/openmrs/values.yaml

```diff
@@ -1,8 +1,12 @@
 replicaCount: 1
 
+global:
+  storageClass: bahmni-efs-sc
```

Comment:
I feel that we can pass an empty value here. And override it from helm-umbrella, because storageClass would be implementation/environment specific.